pkg/migrations/sql/versions/dev/0.1.5-dev.2/1-drop_function_metric_view.sql

```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS SCHEMA_CATALOG.metric_view() CASCADE;
```

Comment:
this file should be pkg/migrations/sql/versions/dev/0.1.5-dev/3-drop_function_metric_view.sql according to our scheme